frontend/src/components/ScheduledEventsTableRow/utils/dateConversion.ts

```diff
@@ -0,0 +1,24 @@
+import { ScheduledEventsTableRowProps } from '../ScheduledEventsTableRow';
+
+export const deriveTimeFromStartDate = (props: ScheduledEventsTableRowProps): string => {
```

Comment:
Nope ... just `startDate` as a parameter... as far as I can tell, that's all you're using from the props.. can even name it neutrally ... `deriveTimeFromDate` ... that way it can be used for any date.